amundsen_common/log/http_header_caller_retrieval.py

```diff
@@ -12,4 +12,4 @@
 class HttpHeaderCallerRetrieval(BaseCallerRetriever):
     def get_caller(self) -> str:
         header_key = flask_app.config.get(CALLER_HEADER_KEY, 'user-agent')
-        return request.headers.get(header_key, 'UNKNOWN')
+        return request.headers.get(header_key) or 'UNKNOWN'
```

Comment:
Possibly a change due to the minor Flask upgrade: headers is a Headers object (not a dict) and the old mypy annotation was broken.